src/www/app/clipboard.ts

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// <reference path='../../types/ambient/clipboard.d.ts'/>
```

Comment:
These directives is the only way to express that this file depends on that type.
Why are they not allowed? What's the suggested alternative?
I'd much rather keep a way to represent the dependency.

Reply:
I forgot you could just use "import" so i did that instead, even though it's now unnecessary (they are implicitly imported everywhere)